recipes/autogluon.common/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "autogluon.common" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autogluon.common-{{ version }}.tar.gz
+  sha256: 4f60d3d1c7c1357b8c681887fdca47951d21115e4c4a0c40122f8b25909cfb24
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<3.10
+    - pip
+  run:
+    - python >=3.7,<3.10
+    - numpy <1.24,>=1.21
+    - pandas !=1.4.0,<1.6,>=1.2.5
+    - boto3
+    - psutil <6,>=5.7.3
+    - setuptools
+
+test:
+  imports:
+    - autogluon.common
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/autogluon.common
+  summary: AutoML for Image, Text, and Tabular Data
+  dev_url: https://github.com/autogluon/autogluon/
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
There's no need to copy the license. Since you reference it here, the version in the upstream repo will be included in the package.

Reply:
Should I remove both line 42 and the LICENSE file or just remove the LICENSE file?

Reply:
Just remove the file here. Keep line 42 - that tells conda where to find the file in the upstream repo.